README.md

```diff
@@ -72,7 +72,7 @@ tags = "country:china,datacenter:asia,account:mass_market"
 logger:count("another_hit", 1, 1, tags)
 ```
 
+## Luarock creation
 
-
-
-
+1. Run `sh scripts/build_rock.sh <version>` where version corresponds to the tag you want to build.  Ex. `sh scripts/build_rock.sh v0.1.0`.
+2. Upload the .src.rock file to the server (TBD)
```

Comment:
TBD because you're going to need an API key to do this...  I'm not sure if we should make a "sifi" account on luarocks.org or what.

Reply:
or just a ngx_lua_datadog "maintainer" key?